cypress/e2e/widgets/openshift-ai.cy.ts

```diff
@@ -1,6 +1,7 @@
 describe('Openshift AI widget', () => {
   beforeEach(() => {
     cy.loadLandingPage();
+    cy.wait(5000);
```

Comment:
@catastrophe-brandon sorry if I missed the conversation - but is there something else on the landing page that we can look for to indicate loading is complete rather than waiting 5 seconds (I'm not convinced on slower envs if this will even be long enough).